frontend/src/components/LandingPage/Features/Features.test.js

```diff
@@ -0,0 +1,26 @@
+import { shallow } from 'enzyme';
+import { screen, render } from '@testing-library/react';
+import Features from './Features'
+import React from 'react';
+
+describe('Test features component', () => {
+    const wrapper = shallow(<Features/>);
+
+    it('Checks if Features component renders', () => {   
+        expect(wrapper.exists()).toBe(true);
+    });
+
+    it('Check if title text renders properly', () => {
+        expect(wrapper.text().includes("Features of our Application")).toBe(true);
+    }); 
+
+    it('Check if cards render text properly', () => {
+        render(<Features/>);
+        let cardTitles = [/Guess Future Stock Prices/i, /Get Stock Information/i, /Get Current News/i]
+        cardTitles.forEach(element => {
+            const textElement = screen.getByText(element);
+            expect(textElement).toBeInTheDocument();
+        });
+    }); 
```

Comment:
The test seems good, but I think you can make it even better by adding all the text of each card. You don't have to test the whole paragraph, but just a little snippet of it.